blog/_posts/2023-07-12-website-overhaul-and-reachability.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: blog-detail
+post-type: blog
+by: Julien Richard-Foy (Scala Center)
+title: Website Overhaul and Improved Reachability
```

Comment:
"Website Overhaul" kind of sounds like the entire website was overhauled, when it was really just the front page. How about "Website: front page overhauled, reachability improved"?